src/util/specific.py

```diff
@@ -60,6 +61,17 @@ def rename_custom_table(
     )
 
 
+def rename_custom_module(cr, old_module_name, new_module_name, report_details=""):
+    rename_module(cr, old_module_name, new_module_name)
+
+    add_to_migration_reports(
+        category="Rename Module",
+        message="The custom module '{old_module_name}' was renamed to '{new_module_name}'. {report_details}".format(
+            **locals()
+        ),
+    )
```

Comment:
```suggestion
def rename_custom_module(cr, old_module_name, new_module_name, report_details="", skip_check=False):
    if skip_check or module_installed(cr, old_module_name):
        rename_module(cr, old_module_name, new_module_name)

        add_to_migration_reports(
            category="Rename Module",
            message="The custom module '{old_module_name}' was renamed to '{new_module_name}'. {report_details}".format(
                **locals()
            ),
        )
```

Avoid writing to migration report if the module is not found in the current db.

Reply:
The module must be renamed even if not installed in many cases.